File: CS/GridColumnHeight.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Model;


namespace GridColumnHeight.Module.Win
{
    public interface IModelListViewHeaderRows : IModelNode
    {
        int HeaderRows { get; set; }
    }

    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class GridColumnHeightWindowsFormsModule : ModuleBase
    {
        public GridColumnHeightWindowsFormsModule()
        {
            InitializeComponent();
        }

        public override void ExtendModelInterfaces(ModelInterfaceExtenders extenders)
        {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<IModelListView, IModelListViewHeaderRows>();
        }
    }
}

File: VB/GridColumnHeight.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp


Namespace GridColumnHeight.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class GridColumnHeightWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Overrides Function GetSchema() As Schema
			Return XAFCommunity.Win.GridHeaderHeightHelper.GetGridHeaderHeightSchema()
			'return base.GetSchema();
		End Function
	End Class
End Namespace
